### ISSUE TITLE
Bizzare placing of plants

### ISSUE THREAD

**Issue by Fixer-007** [HybridDog/hydro#1]
Mod generates those plants in a very weird... linear way, have a look:
* https://i.imgur.com/ICmSEI8.png
* https://i.imgur.com/VlP4dQN.jpg
* https://i.imgur.com/u5qS6iL.png

This needs rework.

**Comment by HybridDog** [comment]
You're right: https://github.com/HybridDog/hydro/blob/master/init.lua#L285
